ldap/servers/slapd/pblock.c

```diff
@@ -2059,6 +2059,13 @@ slapi_pblock_get(Slapi_PBlock *pblock, int arg, void *value)
             (*(int *)value) = 0;
         }
         break;
+    case SLAPI_TASK_MISC:
```

Comment:
Rather than new statements in the case/switch, look around line pblock.c:4280 for individual get/set in the pblock, that are then subject to inlining in compiling. It's better for performance and clarity, as well as avoiding void casting that present type unsafety.

Reply:
That makes more sense alright